bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -604,6 +583,7 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
   void addTimers();
   void startConsensusProcess(PrePrepareMsg* pp, bool isCreatedEarlier);
   void startConsensusProcess(PrePrepareMsg* pp);
+  void postBftExecutionActions(PrePrepareMsg* ppMsg, IRequestsHandler::ExecutionRequestsQueue& requests_for_execution);
```

Comment:
NIT: requests_for_execution should be requestsForExecution

Reply:
fixed